keys/private.pem

```diff
@@ -0,0 +1,8 @@
+-----BEGIN PRIVATE KEY-----
```

Comment:
Never save your private keys in a public repo!!! In a private as well ;)

Please, remove this folder from the repository and regenerate your key.